core/templates/dev/head/exploration_editor/EditorServices.js

```diff
@@ -700,6 +758,58 @@ oppia.factory('explorationStatesService', [
         return (
           validatorsService.isValidStateName(newStateName, showWarnings));
       },
+      getStateContentMemento: function(stateName) {
+        return getStatePropertyMemento(stateName, 'content');
+      },
+      saveStateContent: function(stateName, newContent) {
```

Comment:
Hm, I don't know. I actually did start with set*, but then changed it to save* as this was already standard terminology throughout the codebase for the individual editors. I still lean towards save* for that reason, but do you feel strongly?